internal/flypg/pg.go

```diff
@@ -482,14 +482,14 @@ func (c *PGConfig) setDefaultHBA() error {
 		},
 		{
 			Type:     "host",
-			Database: "all",
+			Database: "replication,all",
```

Comment:
Does this not cover your needs?
https://github.com/fly-apps/postgres-flex/blob/9501aea7e6f0d1e0c06f8f18741530b8c556575a/internal/flypg/pg.go#L469-L482